UI/cmake/macos/entitlements-extension.plist

```diff
@@ -0,0 +1,20 @@
+<!--?xml version="1.0" encoding="UTF-8"?-->
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
+        <true/>
+        <key>com.apple.security.device.camera</key>
+        <true/>
+        <key>com.apple.security.device.audio-input</key>
+        <true/>
+        <key>com.apple.security.cs.disable-library-validation</key>
+        <true/>
+        <key>com.apple.developer.system-extension.install</key>
+        <true/>
+        <key>com.apple.security.application-groups</key>
+        <array>
+            <string>$(TeamIdentifierPrefix)com.obsproject.obs-studio</string>
+        </array>
```

Comment:
Are these keys in any particular order?

Reply:
No order, as they're usually added automatically by Xcode.